### PR TITLE
update ospf modules to fix bugs as software version changes

### PR DESCRIPTION
**Pull request by yuandongx**
##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

lib/ansible/modules/network/cloudengine/ce_ospf_vrf.py
lib/ansible/modules/network/cloudengine/ce_ospf.py
##### ADDITIONAL INFORMATION

CloudEingine has version changes,
And there are few of bugs to fix,more information see code review.

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/network/cloudengine/ce_ospf.py [hunk 1]
```diff
@@ -459,7 +459,7 @@ def get_ospf_dict(self, process_id):
         # get nexthop info
         ospf_info["nexthops"] = list()
         nexthops = root.findall(
-            "data/ospfv2/ospfv2comm/ospfSites/ospfSite/ProcessTopologys/ProcessTopology/nexthopMTs/nexthopMT")
```
Comment (xuxiaowei0512) [inline lib/ansible/modules/network/cloudengine/ce_ospf.py]: To find all netxhops  should not be from root node of which tag name is 'data'.So remove the first tag 'data' and find all chirld node.

File: lib/ansible/modules/network/cloudengine/ce_ospf_vrf.py [hunk 2]
```diff
@@ -1047,7 +1047,7 @@ def get_ospf_info(self):
 
         # get the vpn address family and RD text
         ospf_sites = root.findall(
-            "data/ospfv2/ospfv2comm/ospfSites/ospfSite")
```
Comment (yuandongx) [inline lib/ansible/modules/network/cloudengine/ce_ospf_vrf.py:1050]: 'data' is root node that can not find all.

File: lib/ansible/modules/network/cloudengine/ce_ospf_vrf.py [hunk 3]
```diff
@@ -1063,7 +1063,8 @@ def get_ospf_info(self):
                                              "spfScheduleIntervalType"]:
                         ospf_ele_info[
                             ospf_site_ele.tag] = ospf_site_ele.text
-                self.ospf_info["ospfsite"].append(ospf_ele_info)
+                if ospf_ele_info["processId"] == self.ospf:
```
Comment (yuandongx) [inline lib/ansible/modules/network/cloudengine/ce_ospf_vrf.py:1066]: Find the specified node that is the same as 'self.ospf'

File: lib/ansible/modules/network/cloudengine/ce_ospf.py [hunk 4]
```diff
@@ -442,15 +442,15 @@ def get_ospf_dict(self, process_id):
 
         # get process base info
         root = ElementTree.fromstring(xml_str)
-        ospfsite = root.find("data/ospfv2/ospfv2comm/ospfSites/ospfSite")
+        ospfsite = root.find("ospfv2/ospfv2comm/ospfSites/ospfSite")
```
Comment (yuandongx) [inline lib/ansible/modules/network/cloudengine/ce_ospf.py:445]: Do not find node from root node of which tag name is 'data'.

File: lib/ansible/modules/network/cloudengine/ce_ospf.py [hunk 5]
```diff
@@ -442,15 +442,15 @@ def get_ospf_dict(self, process_id):
...
         if ospfsite:
             for site in ospfsite:
                 if site.tag in ["processId", "routerId", "vrfName"]:
                     ospf_info[site.tag] = site.text
 
         # get Topology info
         topo = root.find(
-            "data/ospfv2/ospfv2comm/ospfSites/ospfSite/ProcessTopologys/ProcessTopology")
+            "ospfv2/ospfv2comm/ospfSites/ospfSite/ProcessTopologys/ProcessTopology")
```
Comment (yuandongx) [inline lib/ansible/modules/network/cloudengine/ce_ospf.py:453]: Same as the previous modification point

**Pull request merged**